scripts/aggregate-translations/write-aggregated-translations.js

```diff
@@ -12,7 +12,23 @@ const writeAggregatedTranslations = (aggregatedMessages, locales, fileSystem, ou
       }
 
       const translationFilePath = path.resolve(outputDir, `${locale}.js`);
-      fileSystem.writeFileSync(translationFilePath, generateTranslationFile(locale, aggregatedMessages[locale]));
+      const messages = aggregatedMessages[locale];
+      const localeRegionSplit = locale.split('-');
+      let mergedMessages = messages;
+
+      // Check if this is a regional locale
+      if (localeRegionSplit.length > 1) {
+        const localeMessages = aggregatedMessages[localeRegionSplit[0]];
```

Comment:
nit: lets call this baseLocaleMessages for clarity

Then can we add an inline comment specifying we are merging on the base locale messages?